src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDynamicTests.cs

```diff
@@ -1370,24 +1370,24 @@ .locals init (C.<>c__DisplayClass11_0 V_0, //CS$<>8__locals0
   IL_02b5:  ldloc.0
   IL_02b6:  ldfld      ""dynamic C.<>c__DisplayClass11_0.dyn""
   IL_02bb:  stloc.2
-  IL_02bc:  ldsfld     ""System.Runtime.CompilerServices.CallSite<System.Func<System.Runtime.CompilerServices.CallSite, object, bool>> C.<>o__11.<>p__12""
+  IL_02bc:  ldsfld     ""System.Runtime.CompilerServices.CallSite<System.Func<System.Runtime.CompilerServices.CallSite, object, bool>> C.<>o__11.<>p__9""
```

Comment:
The changes in this file are just related to the different visit order in the local rewriter changing the order of the fields in the display class. I've manually verified the type IL before and after.